apps/central-scan/backend/src/backup.ts

```diff
@@ -84,21 +83,6 @@ export class Backup {
     await this.addEntry('election.json', electionDefinition.electionData);
     debug('added election.json to backup');
 
-    debug('adding CVRs to backup...');
-    await this.addEntry(
-      CAST_VOTE_RECORD_REPORT_FILENAME,
-      getCastVoteRecordReportStream({
-        electionDefinition,
-        definiteMarkThreshold: this.store.getMarkThresholds().definite,
-        isTestMode: this.store.getTestMode(),
-        resultSheetGenerator: this.store.forEachResultSheet(),
-        batchInfo: this.store.getBatches(),
-        reportContext: 'backup',
-      })
-    );
-
-    debug('added CVRs to backup');
-
```

Comment:
As noted in the commit description, I'll be completely revisiting scanner backups in a follow-up PR. This is just a temporary patch, necessary because I've removed the old CVR export function used here (`getCastVoteRecordReportStream`)